packages/web/docs/package.json

```diff
@@ -9,7 +9,8 @@
     "build": "next build && next-sitemap",
     "dev": "next --turbopack",
     "postbuild": "pagefind --site .next/server/app --output-path out/_pagefind",
-    "validate-mdx-links": "pnpx validate-mdx-links@1.0.6 --files 'src/**/*.mdx'"
+    "validate-mdx-links": "pnpx validate-mdx-links@1.0.6 --files 'src/**/*.mdx'",
+    "generate-security-report": "trivy image -s 'MEDIUM,HIGH,CRITICAL' ghcr.io/graphql-hive/gateway:latest --format template --template '@./scripts/docker-security-report.tpl'"
```

Comment:
this one could be tricky - `latest` can mean a lot of things, and it might get updated without us knowing - any thoughts?

Reply:
This command is just a shortcut to generate the CVE table, so most of the time you will want to run on the latest.

It's more a way to not forget how to generate it :-) With the custom template I've made for markdown ouptput.